.github/security.md

```diff
@@ -1,4 +1,15 @@
-## Security contact information
+# Security Policy
```

Comment:
@sttk I added a "global" security policy to the gulpjs github organization, so we can probably remove this.

Reply:
OK. I'll remove this file.

Reply:
Got it. I'll remove this file.